lib/diagnostic.rb

```diff
@@ -35,8 +45,9 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
-Response.dave = dave
+dave = Person.new("Dave", 32, "Ohio")
+dave.location = "Somerville"
+Response.dave = dave.new
```

Comment:
This will cause an error since `dave` is not a class and doesn't have a constructor.